docs-content/getting-started/fullstack-frameworks/next-js.md

```diff
@@ -28,6 +28,39 @@ all-in-one.
 yarn add @highlight-run/next @highlight-run/react highlight.run
 ```
 
+## Environment Configuration (optional)
+
+> This section is extra opinionated about Next.js constants. It's not for everyone. We like how `zod` and TypeScript work together to validate `process.env` inputs... but this is a suggestion. Do your own thing!
+
+1. Install Zod: `yarn add zod`
+1. Edit `.env` to add your projectID to `NEXT_PUBLIC_HIGHLIGHT_PROJECT_ID`
```

Comment:
should be 2.

Reply:
Markdown supports auto-incrementing lists, but `highlight.io` only supports bulleted lists, so it all looks identical.

I'll number it anyway, cause the list is stable.

![image](https://github.com/highlight/highlight/assets/878947/281a3d53-7bef-42c9-9ddd-a2e13084faf9)